src/Provider/AbstractProvider.php

```diff
@@ -24,16 +24,6 @@
     const ACCESS_TOKEN_UID = null;
 
     /**
-     * @var string HTTP method used to fetch access tokens.
```

Comment:
That doesn't make sense, because the OAuth spec clearly states that the only methods used are `GET` and `POST` as per [Section 3.1](http://tools.ietf.org/html/rfc6749#section-3.1):

> The authorization server MUST support the use of the HTTP "GET"
   method [RFC2616] for the authorization endpoint and MAY support the
   use of the "POST" method as well.